src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,50 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt="Film logo"
```

Comment:
Lets do it this way to be consistent 
```suggestion
          alt="Movie logo"
```